conda/recipes/libcudf/meta.yaml

```
@@ -38,7 +38,7 @@ requirements:
     - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
     - dlpack
     - arrow-cpp-proc * cuda
-    - boost-cpp >=1.72.0
+    - {{ pin_compatible('boost-cpp', max_pin='x.x') }}
```

Would suggest pinning to the patch release as they include the patch release in the SONAME as well.

```suggestion
    - {{ pin_compatible('boost-cpp', max_pin='x.x.x') }}
```

https://abi-laboratory.pro/index.php?view=timeline&l=boost

---

Lol fixed after clicking submit. Marking as resolved